docs/TranslationGuide.md

```diff
@@ -50,29 +47,37 @@ submit the new or updated file as a pull request to Github (see
 [translators guide for Engine]). Contact the Zonemaster Group if
 that does not work.
 
-The translator must always create or update the *LANG.json* and
-the *gui-faq-LANG.md*. The other changes are only done when
+The translator must always create or update the `messages.<LANG>.xlf` and
+the `gui-faq-<LANG>.md`. The other changes are only done when
 a language is added and will be completed by the Zonemaster Group.
 
 
-## LANG.json
+## messages.\<LANG\>.xlf
 
-The JSON structure LANG.json holds all the messages for GUI in respective
+The XML structure LANG.json holds all the messages for GUI in respective
 language, where "LANG" is the language code, e.g. `en.json`.
```

Comment:
"LANG.json" --> "messages.LANG.xlf" or rather "`messages.<LANG>.xlf`"
"en.json" --> "messages.en.xlf"

Reply:
This was meant to be replaced by the line after, I removed this one